src/plugins/PositionPlugin.cpp

```diff
@@ -127,60 +128,77 @@ int32_t PositionPlugin::runOnce()
 {
     NodeInfo *node = nodeDB.getNode(nodeDB.getNodeNum());
 
-    //radioConfig.preferences.position_broadcast_smart = true;
+    // radioConfig.preferences.position_broadcast_smart = true;
 
     // We limit our GPS broadcasts to a max rate
     uint32_t now = millis();
     if (lastGpsSend == 0 || now - lastGpsSend >= getPref_position_broadcast_secs() * 1000) {
 
-        lastGpsSend = now;
+        // Only send packets if the channel is less than 40% utilized.
+        if (airTime->channelUtilizationPercent() < 40) {
 
-        lastGpsLatitude = node->position.latitude_i;
-        lastGpsLongitude = node->position.longitude_i;
+            lastGpsSend = now;
 
-        // If we changed channels, ask everyone else for their latest info
-        bool requestReplies = currentGeneration != radioGeneration;
-        currentGeneration = radioGeneration;
+            lastGpsLatitude = node->position.latitude_i;
+            lastGpsLongitude = node->position.longitude_i;
+
+            // If we changed channels, ask everyone else for their latest info
+            bool requestReplies = currentGeneration != radioGeneration;
+            currentGeneration = radioGeneration;
+
+            DEBUG_MSG("Sending pos@%x:6 to mesh (wantReplies=%d)\n", node->position.pos_timestamp, requestReplies);
+
+            sendOurPosition(NODENUM_BROADCAST, requestReplies);
+
+        } else {
+            DEBUG_MSG("Channel utilization is >50 percent. Skipping this opportunity to send.\n");
```

Comment:
Debug message states it's above 50 percent while it checks if it is above 40%.

Reply:
I’ll fix the typo thank you